test/blackbox/pushpull_running_dedupe.bats

```diff
@@ -119,15 +119,19 @@ function teardown_file() {
     sed -i 's/false/true/g' ${zot_config_file}
 
     zot_serve ${ZOT_PATH} ${zot_config_file}
+
+    # give time to repoDB to parse storage
+    sleep 10
```

Comment:
As far as I know zot won't start serving requests until repoDB finishes parsing the storage.

Reply:
yes, that should be the preferred way IMO.
Do a curl GET /v2/ check, sleep loop until 200.

Reply:
then I'll increase the timeouts of curl, it's not enough for this test